lib/account_deleter.rb

```diff
@@ -69,10 +68,6 @@ def delete_standard_person_associations
     end
   end
 
-  def disconnect_contacts
```

Comment:
In this case: tests with different combinations of contacts. Without `disconnect_contacts` it fails when you have double sided contacts, for me it failed without `.reload`. So something that covers that.

But I just tested, when I remove the `.reload` it now also fails with the updated DataGenerator, so it's maybe already enough. I just want to make sure that we don't remove the `.reload`  or `disconnect_contacts` and something is broken again without a failing test.

But I'll have a closer look at this again tomorrow. #7538 should fix what I broke and as it looks it also makes the tests failing if I break something again, so for now that's good enough ;) 😴